owl/SOMA-OBJ.owl

```diff
@@ -1,16 +1,16 @@
-Prefix(:=<http://www.ease-crc.org/ont/SOMA-OBJ.owl#>)
```

Comment:
Can you save this as OWL Functional syntax as opposed to rdf/xml? Maybe that will reduce the size of the diff I have to look at.

Reply:
As with the SOMA-ACT file, there's no reasonable way to get a review for a diff this size, nor is the diff itself that informative when most of it comes from format differences.